tests/FlatBuffers.Test.Swift/Tests/FlatBuffers.Test.SwiftTests/FlatBuffersMonsterWriterTests.swift

```diff
@@ -186,5 +186,33 @@ class FlatBuffersMonsterWriterTests: XCTestCase {
         } else {
             XCTAssertEqual(monster.testarrayofbools.isEmpty, false)
         }
+        
+        var m: MyGame.Example.MonsterT? = monster.unpack()
+
+        var builder = FlatBufferBuilder()
+        builder.finish(offset: Monster1.pack(&builder, obj: &m))
+        let mo = Monster1.getRootAsMonster(bb: builder.sizedBuffer)
+        XCTAssertEqual(mo.hp, 80)
```

Comment:
instead, can you simply call the existing test code for the non-object-API here (on `mo`)? If anything, some test accesses specifically to the object API (on `m`) would be good.